docs/topics/async.txt

```diff
@@ -260,3 +260,10 @@ thread and thus is fully compatible with async mode. Note that sync code will
 always be in a *different* thread to any async code that is calling it, so you
 should avoid passing raw database handles or other thread-sensitive references
 around.
+
+.. versionchanged:: 3.1.3
+
+    ``asgiref`` version 3.3.0 changed the default value of the
+    ``thread_sensitive`` to ``True``. This is a safer default, and in many
+    cases interacting with Django the correct value, but be sure to evaluate
+    uses of ``sync_to_async()`` if updating ``asgiref`` from a prior version.
```

Comment:
It's not a change in Django so I would change it to a `warning` and move below [L234](https://github.com/django/django/pull/13586/files#diff-5a966002c75ed9d9db0802c9ad49222a8b4897a7ad6fd68938a5e2d7fb195107R243).